src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newObj = {};
+  const result = [];
+
+  Object.assign(newObj, state);
```

Comment:
You can just write `const newObj = { ...state };`

Reply:
agree